### ISSUE TITLE
Allow Cube model to have multiple strands

### ISSUE THREAD

**Issue by kevinsaucier** [xLightsSequencer/xLights#4621]
I use the Cube model for pixel sticks, which has worked great in previous years.  This year, I'm adding an additional prop on top of a number of sticks but there is no easy way to insert that prop before/after a stick so I have to run long extension cords between each of the toppers or try to chain them before/after/between the cube models.  I've tried exporting/importing as custom but it really doesn't make anything easier and then I'm using a custom model instead of native, which I know has drawbacks.  I know I could also set up 50 individual cube models but it would take forever and be a nightmare to manage.  

Anyone have a better idea?  If not, (here's the enhancement request 😁) is it possible/feasible to allow the cube model to have multiple strands?   Thanks!

![image](https://github.com/xLightsSequencer/xLights/assets/25715814/3402c749-8f3f-48dd-b458-6359d49de09f)

**Comment by cybercop23** [comment]
@kevinsaucier wait...... you want multiple strands for the cube so you can add the toppers? So the cube is on string 1, and toppers are on string 2?

**Comment by kevinsaucier** [comment]
Well, as I think about it, this probably doesn't work.  My original thought was that I'd like to be able to, in visualizer, stick a topper model in the middle of stick 1 and stick 2.  That doesn't really work with multiple strings because then I'd need those strings to map to their own outputs on the controller, right?  

I guess what I'd like to see is the ability to have a stick model with x sticks on output 1, then a topper, then a stick model of x more sticks after the topper, then another topper, repeat repeat, all on output 1.  And all without creating 6 cube models per row.  

I was hoping to do that with a single cube model per row, but I don't think that's really doable at this point, unless I maybe create a bunch of shadow models, which is pretty much the same amount of work.

That make sense and still sound undoable? 😁

**Comment by cybercop23** [comment]
Yes, and that's what I thought. Not doable. Think of a mega tree with a star. The star can be after the last string, but not in the middle. Well... technically it will work for FPP-based controllers since they create virtual strings, but your topper may not be anywhere near an end/start string.
You can do a receiver in the middle of the cube, to the tentacles where you want the toppers.

**Comment by kevinsaucier** [comment]
Technically, it would be doable, if xlights allowed the 'strings' to be on the same output.  Not sure what the technical reason is but that seems like it should be doable.  Seems like models would be better off being broken into 'sections' instead of strings, to allow for sectioning a model and inserting other models in the middle.  Would be really handy for your example of mega/mini trees and stars.  One can dream.....  😁
